service/src/main/java/com/ge/predix/acs/config/PolicyEvaluationCacheConfig.java

```diff
@@ -17,22 +23,27 @@
     @Value("${ENABLE_CACHING:false}")
     private boolean cachingEnabled;
 
-    @Value("${ENABLE_CIRCUIT_BREAKER:false}")
-    private boolean hystrixCircuitBreakerEnabled;
+    @Autowired
+    private Environment environment;
 
     @Bean
-    public PolicyEvaluationCacheCircuitBreaker cache() {
+    public PolicyEvaluationCache cache() {
         if (!this.cachingEnabled) {
-            LOGGER.info("Caching is disabled.");
-            return new BasicPolicyEvaluationCacheCircuitBreaker();
+            if (LOGGER.isDebugEnabled()) {
+                LOGGER.debug("Caching disabled");
+            }
+            return new NonCachingPolicyEvaluationCache();
+        }
+        List<String> activeProfiles = Arrays.asList(this.environment.getActiveProfiles());
+        if (activeProfiles.contains("redis") || activeProfiles.contains("cloud-redis")) {
+            if (LOGGER.isDebugEnabled()) {
+                LOGGER.debug("Redis caching enabled");
```

Comment:
Since we are going to have several caches for various purposes now, change the message to "Redis caching enabled for policy evaluation."